### ISSUE TITLE
Add support for running rumdl fmt on github action

### ISSUE THREAD

**Issue by remyleone** [rvben/rumdl#810]
### Problem or motivation

It would be helpful to be able to run `rumdl fmt` using the official github action. I'm doing it because I would like to check whether a file is well formatted. At the moment there is only a check that is available.

### Proposed solution

Have a way to configure the action to be able to also run it as fmt

### Alternatives considered

install it manually, run the binary with the fmt way.

### Feature category

Other

### Example markdown

```markdown

```

### Contribution

- [ ] I would be willing to submit a PR to implement this feature

**Comment by rvben** [comment]
Released in v0.2.54, and `rvben/rumdl@v0` carries it. A `command` input selects what runs:

```yaml
- uses: rvben/rumdl@v0
  with:
    command: fmt-check
```

- `check` is the default and is unchanged.
- `fmt-check` runs `rumdl fmt --check`, which prints a diff of what would be reformatted and fails the workflow without touching the files. That is the one you want for checking whether a file is well formatted.
- `fmt` runs `rumdl fmt`, which rewrites the files in the workspace.

`config`, `report-type`, `fail-on-error`, `output-file` and `args` apply to all three. `install-only` from #808 ignores `command` along with the other lint inputs, and since PATH export is now unconditional you can also just call `rumdl fmt` yourself from a later step.
